### DISCUSSION TITLE
How do I get started??

### DISCUSSION THREAD

**Discussion by dharunnarayan31** [google-deepmind/penzai#12] [Q&A] [answered]
How do I get started with this???

**Comment by shiftbug** [comment, marked as answer]
> How do I get started with this???

The first thing would likely be to review this document: https://penzai.readthedocs.io/en/stable/notebooks/how_to_think_in_penzai.html

Or here: https://penzai.readthedocs.io

**Reply by danieldjohnson** [reply]
Yep, the documentation @shiftbug linked should be the best way to get started!

You can also click the "Open in Colab" links from any of the tutorial notebooks to make a copy of the Colab notebook and run the commands yourself.
